src/simulator/base.py

```diff
@@ -0,0 +1,27 @@
+from object import Object, ObjectId
+from position import Position
+from shape import Shape
+from object_registry import ObjectRegistry
+from robot import Robot
+import numpy as np
+
+RadioID = str  # type aliasing, since we might change RadioID composition
+
+
+class Base(Object):
+    def __init__(self, oid: ObjectId, name: str, shape: Shape, position: Position,
+                 owner_object: Object, registry: ObjectRegistry, rid: RadioID, error: int):
```

Comment:
From my understanding, `error` is the standard deviation of the normal distribution, so shouldn't it be `float` instead of `int`?

Reply:
you're right. good point.